src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -926,8 +926,9 @@ def _fill_ftp_publishing_url(cmd, webapp, resource_group_name, name, slot=None):
 
 
 def _format_fx_version(custom_image_name, container_config_type=None):
-    custom_image_name = custom_image_name.lower()
-    custom_image_name = custom_image_name.replace("https://", "").replace("http://", "")
+    lower_custom_image_name = custom_image_name.lower()
+    if "https://" in lower_custom_image_name or "http://" in lower_custom_image_name:
```

Comment:
Can the if statement be omited?

Reply:
@qwordy since there are scenarios in which custom_image_name contains a base64 encoded string (which is actually what is broken right now, see #14208 ), we only want to convert to lower if it contains http:// or https://